ibm/mas_devops/roles/ocp_deprovision/tasks/providers/fyre.yml

```diff
@@ -41,7 +42,7 @@
     - _cluster_exist.json is defined
     - _cluster_exist.json.owning_user is defined # when there's cluster owner, it means the cluster exists thus we can deprovision it
   uri:
-    url: "https://ocpapi.svl.ibm.com/v1/ocp/{{cluster_name}}"
+    url: "https://ocpapi.svl.ibm.com/v1/ocp/{{ cluster_name }}?site={{ fyre_site }}"
```

Comment:
To delete also.

Reply:
Same as https://github.com/ibm-mas/ansible-devops/pull/1085/files#r1383330877

<img width="1004" alt="image" src="https://github.com/ibm-mas/ansible-devops/assets/15021471/16409cbe-e706-4e77-b4ba-71fa5dd7b209">